test/rules/best-practises/no-empty-blocks.js

```diff
@@ -123,4 +123,99 @@ describe('Linter - no-empty-blocks', () => {
     const labelIndex = Math.floor(lastItemIndex / 5) * 4
     return items[labelIndex]
   }
+
+  describe('configuration options', () => {
+    it('should raise warning for empty modifier with _ when allowEmptyModifiers is false', () => {
+      const code = contractWith(`
+        modifier onlyOwner  {
+          _;
+        }
+      `)
+
+      const report = linter.processStr(code, {
+        rules: {
+          'no-empty-blocks': ['warn', { allowEmptyModifiers: false }],
+        },
+      })
+
+      assertWarnsCount(report, 1)
+      assertErrorMessage(report, 'Code contains empty blocks')
+    })
+
+    it('should raise warning for completely empty modifier', () => {
```

Comment:
this I believe is a nice to have, but if implementation ends up being simpler by ignoring it, we can live with it